pkg/client/unversioned/remotecommand/remotecommand.go

```diff
@@ -194,12 +194,19 @@ func (e *Streamer) doStream() error {
 		if err != nil {
 			return err
 		}
-		defer remoteStdin.Reset()
```

Comment:
Hmm, assuming I merged and rebuilt everything locally correctly, trying to exit an interactive shell is hanging. Grrr....